packages/terra-dropdown-button/src/_Dropdown.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import FocusTrap from 'focus-trap-react';
+import Hookshot from 'terra-hookshot';
+import DropdownList from './_DropdownList';
+
+const propTypes = {
+  /**
+   * Callback to tell the parent it should close the dropdown
+   */
+  requestClose: PropTypes.func.isRequired,
+  /**
+   * Ref to attach the dropdown to
+   */
+  targetRef: PropTypes.func.isRequired,
+  /**
+   * Whether or not the dropdown should be open
+   */
+  isOpen: PropTypes.bool.isRequired,
+  /**
+   * The Options that should be displayed in the dropdown
+   */
+  children: PropTypes.node.isRequired,
+  /**
+   * @private
+   * Width to set the dropdown to. Used when `isBlock` is true. Unset means to autosize.
+   */
+  width: PropTypes.string,
```

Comment:
This could be a question for the UX. The UX callout shows that the width of the dropdown extends up until the width of the button but the example on the site does not. Is the callout showing the default Dropdown or is it showing how it is when `isBlock` is `true`?

Callout: 
<img width="335" alt="Screen Shot 2019-07-03 at 10 31 09 AM" src="https://user-images.githubusercontent.com/10090859/60565068-b5527c00-9d7f-11e9-8044-d90395ad9915.png">


Site Example: 
<img width="116" alt="Screen Shot 2019-07-03 at 10 41 33 AM" src="https://user-images.githubusercontent.com/10090859/60565056-a5d33300-9d7f-11e9-8ac1-d4afac7f065b.png">

Reply:
I think that is just a bad example, from the UX callouts:

> The menu is not tied to the size of the button. It may be larger or smaller than the button, except when using Block Style Button.